packages/pay/exports/react.ts

```diff
@@ -80,6 +80,7 @@ interface UsePayParameters {
  * @param {UsePayParameters} [parameters] - Optional configuration for the hook, including success and error callbacks.
  * @returns {UsePayReturn} An object containing the payment state and actions.
  */
+/** https://docs.reown.com/appkit/react/core/hooks#usepay */
```

Comment:
Better to explain a bit the function with JSDoc structure. See: https://github.com/reown-com/appkit/blob/main/packages/controllers/src/controllers/OptionsController.ts#L36-L40